### PR TITLE
chore(tiering): More advanced tiering tests

### PR DESCRIPTION
**Pull request by dranikpg**
* Add memory usage test
* Add mixed APPEND call test

### CODE REVIEW

**Comment by dranikpg** [comment]
Running without slow first to see how long it takes on CI, maybe we can add a separate drive to the runner with proper io ops on

**Comment by dranikpg** [comment]
Didn't even make it into the top-10 slowest 🤔 suspiciously fast 🤔

**Comment by kostasrim** [comment]
> Didn't even make it into the top-10 slowest 🤔 suspiciously fast 🤔

df

File: tests/dragonfly/tiering_test.py [hunk 1]
```diff
@@ -1,39 +1,81 @@
...
-        for i = 1, 100 do
-            redis.call('SET', 'k' .. i, string.rep('a', 3000))
-        end
+async def test_basic_memory_usage(async_client: aioredis.Redis):
     """
+    Loading 1GB of mixed size strings (256b-16kb) will keep most of them on disk and thus RAM remains almost unused
+    """
+
+    seeder = StaticSeeder(
+        key_target=200_000, data_size=2048, variance=8, samples=100, types=["STRING"]
```
Comment (romange) [inline tests/dragonfly/tiering_test.py:23]: Is the comment above correct?

**Review by romange — APPROVED** [review]

File: tests/dragonfly/tiering_test.py (hunk 1 above)
Comment (dranikpg) [reply tests/dragonfly/tiering_test.py:23]: Yes, the table takes up a lot of space, real memory usage is 1gb

**Pull request merged**